src/Components/InputBox.jsx

```diff
@@ -0,0 +1,40 @@
+import React, { Component } from "react";
+
+export default class InputBox extends Component {
+  constructor(){
+    super();
+    this.state = {
+        nodeValue: 0
+    }
+}
+
+
+  handleNodeValue = (event) => {
+    this.setState({
+        nodeValue: event.target.value
+        
+    })
```

Comment:
You probably need to take a callback for this in via props, as it is no one outside this component will ever be able to know what the value is set to.